warehouse/redshift.go

```diff
@@ -135,22 +135,27 @@ func (rs *Redshift) LoadToWarehouse(file string) error {
 		log.Printf("Failed to upload file %s to s3: %s", file, err)
 		return err
 	}
-	defer rs.RemoveS3Object(s3obj)
 
-	rs.conn, err = rs.MakeRedshfitConnection()
-	if err != nil {
-		return err
-	}
-	defer rs.conn.Close()
+	if rs.conf.S3.S3Only {
+		log.Printf("Config flag S3Only is on, skipping load to Redshift")
+	} else {
+		defer rs.RemoveS3Object(s3obj)
```

Comment:
I considered adding a second flag that allows you to turn off file deletion even if you're loading into Redshift.  Not sure if anyone would need that unless they were debugging, though.

Reply:
agreed